### PR TITLE
made chat screen

### LINKED ISSUE #108: Adding Chat Screen
**Is your feature request related to a problem? Please describe.**
I would like to design the chat screen ui for the app, plz let me work on this issue @infiniteoverflow

### PR DESCRIPTION
**Pull request by anotherwebguy**
### Related Issue
- Fixes #108  .

### Proposed Changes
- i've made the layout of the chat screen
- and have also designed conversation screen

### Additional Information
- For now i've implemented the features u asked for @infiniteoverflow 
- the message that we send appears on the conversation screen as the first message by the user itself and the 2nd msg by the people with whom we are talking for testing
- have aslo implemented emoji picker

### Checklist
- [x] Tested
- [x] No Conflicts
- [x] Change In Code

### Screenshots (Optional)

https://user-images.githubusercontent.com/66346161/115856371-08f8f280-a44a-11eb-9c4a-89da5a954551.mp4

### CODE REVIEW

**Review by pratyushmp — APPROVED** [review]

File: lib/Screens/chat_screen.dart [hunk 1]
```diff
@@ -1,12 +1,211 @@
...
+            },
+          ),
+        ],
       ),
     );
+
+    return Scaffold(
+        appBar: PreferredSize(
+          preferredSize: Size.fromHeight(70),
+          child: new Container(
```
Comment (pratyushmp) [inline lib/Screens/chat_screen.dart:72]: The container doesn't fit the full width of the device.

**Review by pratyushmp — CHANGES_REQUESTED** [review]

File: lib/Screens/chat_screen.dart (hunk 1 above)
Comment (anotherwebguy) [reply lib/Screens/chat_screen.dart:72]: 🤔it takes the whole width in my screen... anyways i''ll give it mediaquery whole width and make the commit soon by today

Comment (anotherwebguy) [reply lib/Screens/chat_screen.dart:72]: made the changes u asked for :)

**Comment by anotherwebguy** [comment]
i've made the changes ..... hope u'll merge it soon

**Pull request merged**